src/librustc/hir/map/mod.rs

```diff
@@ -928,6 +928,8 @@ pub fn map_decoded_item<'ast, F: FoldOps>(map: &Map<'ast>,
                                           ii: InlinedItem,
                                           fold_ops: F)
                                           -> &'ast InlinedItem {
+    let _ignore = map.forest.dep_graph.in_ignore();
```

Comment:
I would either add a comment here why this is needed, or just use `&map.forest.krate` instead of `map.krate()` below, so we don't register a read.